src/EnergyPlus/SingleDuct.cc

```diff
@@ -1963,6 +1943,16 @@ namespace SingleDuct {
                                 sd_airterminal(SysNum).SysName);
         }
 
+        // common report variable for all single duct air terminals
+        for ( int sdIndex = 1; sdIndex <= NumSDAirTerminal; ++sdIndex) {
+            SetupOutputVariable("Zone Air Terminal Outdoor Air Volume Flow Rate",
+                OutputProcessor::Unit::m3_s,
+                sd_airterminal(sdIndex).OutdoorAirFlowRate,
+                "System",
+                "Average",
+                sd_airterminal(sdIndex).SysName );
+        }
+
```

Comment:
Common place output variable setup for all single duct air terminal in in SingleDuct.cc module.